modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/try.kt

```diff
@@ -87,6 +87,15 @@ interface TryFunctor : Functor<ForTry> {
     fix().map(f)
 }
 
+@extension
+interface TryApply : Apply<ForTry> {
+  override fun <A, B> TryOf<A>.ap(ff: TryOf<(A) -> B>): Try<B> =
+          fix().ap(ff)
```

Comment:
I believe in Arrow we use 2 spaces but we are in the process of adding Ktlint // cc @aballano . You can leave or change this but eventually, it's gonna get reformated.

Reply:
Yeap, I couldn't find any tool to automatically format the code and I decided to left the formatting issue as is until I get the answers about the docs and the extensions 😃 I'll format the code properly before finishing this PR. Don't worry 😃

Reply:
Still working on it! #1405